Ice/MenuBar/Appearance/MenuBarOverlayPanel.swift

```diff
@@ -266,10 +266,6 @@ final class MenuBarOverlayPanel: NSPanel {
             Logger.overlayPanel.debug("Menu bar is hidden by system. \(actionMessage)")
             return nil
         }
-        guard !appState.isActiveSpaceFullscreen else {
-            Logger.overlayPanel.debug("Active space is fullscreen. \(actionMessage)")
-            return nil
-        }
```

Comment:
This prevents the secondary panel from updating, leading to incorrect positioning.